crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
NIST SP 800-90A on pg 49 says that the entropy (which we call randomness) must match the security strength as defined in NIST SP 800-57.  We use AES-128.  According to the '57 document in Table 2 on page 53, the security strength of that cipher is 128bits.  Therefore 128 bits of randomness is all that's needed.  Make sense?